msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Specify App Runtime Requirements for Microsoft 365
```

Comment:
We should retain "Microsoft 365" for SEO purposes--otherwise it sounds like it only applies to Teams-only apps.

Reply:
@erikadoyle, according to [this](https://review.learn.microsoft.com/en-us/help/platform/seo-meta-title?branch=main&branchFallbackFrom=main%22%20%5Ct%20%22_blank), meta titles can be max 65 characters. Since we append "- Teams | Microsoft Learn" to all meta titles by default, we are only left with 39 characters. Hence, I suggested we remove "for Microsoft 365".